src/components/usage-guidelines/usage-guidelines.tsx

```diff
@@ -1,13 +1,17 @@
 import React, { useMemo } from 'react';
-import PropTypes from 'prop-types';
 import styles from './usage-guidelines.module.scss';
+import { ElementContent } from '../../types';
 
-const UsageGuidelines = ({ guidelines }) => {
+interface UsageGuidelinesProps {
+  guidelines: Array<string | ElementContent>;
```

Comment:
ElementContent already contains string in the definition
```suggestion
  guidelines: Array<ElementContent>;
```